terra-batch-libchart/templates/_wds.tpl

```diff
@@ -40,6 +40,8 @@ spec:
               value: "{{ .Values.postgres.port }}"
             - name: SWAGGER_BASE_PATH
               value: "{{ .Values.env.swaggerBasePath }}"
+            - name: WORKSPACE_ID
+              value: "{{ .Values..persistence.workspaceManager.workspaceId }}"
```

Comment:
Remove the `..` 

Should be: 

`value: "{{ .Values.persistence.workspaceManager.workspaceId }}"`